### PR TITLE
kaish installs now in ~/.kaish/dist. Added packages installation

### PR DESCRIPTION
**Pull request by D1reCT**

### CODE REVIEW

File: modules/operation/conf/main.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+default: &default
+  operation:
+    package_repository: github.com/helios-technologies/kaigara-%s.git
```
Comment (mod) [inline modules/operation/conf/main.yml]: base url must be github.com/%s/kaigara_%s.git

a package can be on mod/kaigara_ftp.git

File: modules/operation/impl/kaigara_package.rb [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    # @param [NetSSH::Connection] ssh SSH connection
+    # @param [String] command
+    def exec_cmd(ssh, command)
+      result = {
+          stdout: '',
+          stderr: '',
+          code: nil,
+          signal: nil
+      }
+
```
Comment (mod) [inline modules/operation/impl/kaigara_package.rb]: no this command is not a remote install,
it must install locally.

File: modules/operation/operation.rb [hunk 3]
```diff
@@ -47,5 +49,26 @@ def exec
...
       package.run!
     end
+
+    # Install a package
+    # =>
+    desc 'install <package> <uri>', 'Install a kaigara package'
+    def install(package, uri)
+      pkg = KaigaraPackage.new(uri, package)
+      begin
+        config = YAML.load_file(File.dirname(__FILE__) + '/conf/main.yml')
```
Comment (mod) [inline modules/operation/operation.rb]: please move this into a lazy loaded class
Kaigara.config can do the job.

File: modules/operation/operation.rb [hunk 4]
```diff
@@ -47,5 +49,26 @@ def exec
       say "Executing #{package.name}#{"/#{package.version}" if package.version}...", :yellow
       package.run!
     end
+
+    # Install a package
+    # =>
```
Comment (mod) [inline modules/operation/operation.rb]: Install package locally not remotely.
The only remote commands are : 
- operation bootstrap

File: modules/operation/operation.rb (hunk 3 above)
Comment (D1reCT) [inline modules/operation/operation.rb]: what is lazy loaded class?

**Pull request closed without merging**